src/button/index.tsx

```diff
@@ -19,21 +19,26 @@ const StyledButton = styled(Button)`
     text-decoration: none;
     cursor: pointer;
     font-family: inherit;
-    min-height: ${componentSizes.buttonIconSize};
+    height: ${componentSizes.buttonIconSize};
     min-width: ${componentSizes.buttonIconSize};
     max-width: 100%;
     overflow: hidden;
     user-select: none;
     align-items: center;
     transition: max-width ${transition.duration.fast} ease-in,
       background-color ${transition.duration.base} ease-in;
+    white-space: nowrap;
   }
 
   & svg,
   & img {
     flex-shrink: 0;
   }
 
+  &:not(.kirk-button-bubble) svg {
+    margin-right: ${space.s};
+  }
```

Comment:
I don't think we should do this :/ having exceptions inside a button component based on what icon is inside is defeating the purpose of normalizing styles.

Reply:
I inverted the css logic ;-)